python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -482,6 +484,29 @@ def addMissingKeys(self, dataId, butler, missingKeys=None, calibName=None):
             else:
                 raise RuntimeError("No unique lookup for %s: %s" % (k, v))
 
+    def updateMetadata(self, combined, exposureTime, darkTime=None):
+        """Update the metadata"""
+
+        if darkTime is None:
+            darkTime = exposureTime     # avoid warning messages when using calibration products
+
+        visitInfo = afwImage.makeVisitInfo(exposureTime=exposureTime, darkTime=darkTime)
+        md = combined.getMetadata()
+
+        try:
+            afwImage.setVisitInfoMetadata(md, visitInfo)
+        except Exception as e:
+            #
+            # should not be needed with pybind11
+            #
+            # Convert a PropertSet to a PropertyList
+            #            
+            _md = dafBase.PropertyList()
+            _md.combine(md);  md = _md
```

Comment:
Apparently, multiple statements on one line are contrary to the updated coding standards.